File: storio-content-resolver-annotations-processor/src/main/java/com/pushtorefresh/storio/contentresolver/annotations/processor/introspection/StorIOContentResolverColumnMeta.java
package com.pushtorefresh.storio.contentresolver.annotations.processor.introspection;

import com.pushtorefresh.storio.common.annotations.processor.introspection.JavaType;
import com.pushtorefresh.storio.common.annotations.processor.introspection.StorIOColumnMeta;
import com.pushtorefresh.storio.contentresolver.annotations.StorIOContentResolverColumn;

import org.jetbrains.annotations.NotNull;

import javax.lang.model.element.Element;

public class StorIOContentResolverColumnMeta extends StorIOColumnMeta<StorIOContentResolverColumn> {

    public StorIOContentResolverColumnMeta(
            @NotNull Element enclosingElement,
            @NotNull Element element,
            @NotNull String fieldName,
            @NotNull JavaType javaType,
            @NotNull StorIOContentResolverColumn storIOColumn) {
        super(enclosingElement, element, fieldName, null, javaType, storIOColumn);
    }
}

File: storio-content-resolver-annotations-processor/src/main/kotlin/com/pushtorefresh/storio/contentresolver/annotations/processor/introspection/StorIOContentResolverColumnMeta.kt
package com.pushtorefresh.storio.contentresolver.annotations.processor.introspection

import com.pushtorefresh.storio.common.annotations.processor.introspection.JavaType
import com.pushtorefresh.storio.common.annotations.processor.introspection.StorIOColumnMeta
import com.pushtorefresh.storio.contentresolver.annotations.StorIOContentResolverColumn

import javax.lang.model.element.Element

class StorIOContentResolverColumnMeta(
        enclosingElement: Element,
        element: Element,
        fieldName: String,
        javaType: JavaType,
        storIOColumn: StorIOContentResolverColumn,
        getter: String? = null,
        setter: String? = null)
    : StorIOColumnMeta<StorIOContentResolverColumn>(
        enclosingElement,
        element,
        fieldName,
        javaType,
        storIOColumn,
        getter,
        setter)